internal/storage/main.go

```diff
@@ -40,6 +44,19 @@ type ActionEvent struct {
 	Meta      MetaData   `json:"meta,omitempty"`
 }
 
+func (ae *ActionEvent) ExportMetrics(promStorage *prometheus.GaugeVec) {
+	for _, claim := range ae.Data.Claims {
+		promStorage.With(prometheus.Labels{
+			"type":        ae.Type,
+			"eventtype":   ae.EventType,
+			"claimenv":    strconv.Itoa(claim.Environment),
+			"claimpvc":    claim.PersisteStorageClaim,
+			"project":     ae.Meta.Project,
+			"environment": ae.Meta.Environment,
+		}).Set(float64(claim.BytesUsed))
```

Comment:
I'm not sure how relevant exporting the `type` and `eventType` are, as this is used only by Lagoon to determine how to handle the message. I don't see any reason to capture either of these in the metrics